vm/rust/src/lib.rs

```diff
@@ -60,7 +60,7 @@ extern "C" {
     fn JunoAppendTrace(reader_handle: usize, json_trace: *const c_void, len: usize);
     fn JunoAppendResponse(reader_handle: usize, ptr: *const c_uchar);
     fn JunoAppendActualFee(reader_handle: usize, ptr: *const c_uchar);
-    fn JunoAppendDataGasConsumed(reader_handle: usize, ptr: *const c_uchar);
+    fn JunoAppendDataGasConsumed(reader_handle: usize, ptr: *const c_uchar, ptr: *const c_uchar);
```

Comment:
I think we should rename the function because it's no longer about data gas, but rather about gas consumed (L1 and L1 data gas). So, JunoAppendGasConsumed seems more appropriate, in my opinion.